tests/src/CompilerTest.php

```diff
@@ -767,4 +794,35 @@ private function verify($scenarioJson, $method, $arguments, $mysql, $phpInput, $
             );
         }
     }
+
+    private function verifyException($scenarioJson, $method, $arguments, $code, $data)
+    {
+        // cinnabari stuff
+        $lexer = new Lexer();
+        $parser = new Parser();
+        $schema = new Schema(json_decode($scenarioJson, true));
+        $compiler = new Compiler($schema);
+
+        $tokens = $lexer->tokenize($method);
+        $request = $parser->parse($tokens);
```

Comment:
Much of this is duplicated in the "verify" function. We could improve maintainability if we do these steps once in a helper function.